cola/qtutils.py

```diff
@@ -515,25 +515,25 @@ def help_icon():
 
 
 def add_icon():
-    return icon('add.svg')
+    return QtGui.QIcon.fromTheme("list-add", icon('add.svg'))
 
 
 def remove_icon():
-    return icon('remove.svg')
+    return QtGui.QIcon.fromTheme("list-remove", icon('remove.svg'))
 
 
 def open_file_icon():
-    return icon('open.svg')
+    return QtGui.QIcon.fromTheme("document-open", icon('open.svg'))
 
 
 def options_icon():
     """Return a standard open directory icon"""
-    return icon('options.svg')
+    return QtGui.QIcon.fromTheme("configure", icon('options.svg'))
```

Comment:
Sorry for the style nit, but can you please use single-quotes here so that it's consistent with the rest of the line?

In a follow-up commit, I might be tempted into refactoring theme_icon() to allow for this kind of invocation by allowing for an optional 2nd argument.  That way these calls can all use theme_icon().  Don't worry about that now, though, this'll be fine once the quote thing is fixed-up.